app/src/main/java/com/eventyay/organizer/core/auth/start/StartViewModel.java

```diff
@@ -37,6 +37,8 @@
     ;
     private final SingleEventLiveData<Boolean> isEmailRegistered = new SingleEventLiveData<>();
     private final MutableLiveData<EmailRequest> emailRequestModel = new MutableLiveData<>();
+    private final MutableLiveData<Boolean> showWarningDialog=new MutableLiveData<>();
```

Comment:
Proper style

Reply:
In this Should the name of the live data be changed or the casing of the variable?

Reply:
@iamareebjamal  can you have a look at it